infra/aws/terraform/prow-build-cluster/prow.tf

```diff
@@ -25,3 +25,11 @@ resource "aws_iam_openid_connect_provider" "k8s_prow" {
   client_id_list  = ["sts.amazonaws.com"]
   thumbprint_list = ["08745487e891c19e3078c1f2a07e452950ef36f6"]
 }
+
+#resource "aws_iam_openid_connect_provider" "k8s_infra_prow" {
```

Comment:
Please deploy the AWS diff in https://github.com/kubernetes/k8s.io/pull/7205

Sorry about the diff noise.

Reply:
This resource exists, please import it too

Reply:
@upodroid No worries. I already saw this PR, but I need a clarification:

I see `gke_prow_cluster` and `gke_utility_cluster` in the `prow.tf` file, but `iam.tf` contains changes for `gke_utility_cluster[0].arn` and `k8s_infra_prow[0].arn`

When I first ran the plan, it was removing k8s-infra-prow, so yes, it's already created, but not mentioned in the `prow.tf`

Reply:
It's imported:

```bash
terraform import \
  -var-file=./terraform.tfvars \
  -var-file=./terraform.prod.tfvars \
  aws_iam_openid_connect_provider.k8s_infra_prow\[0\] \
  arn:aws:iam::468814281478:oidc-provider/container.googleapis.com/v1/projects/k8s-infra-prow/locations/us-central1/clusters/prow
```